upgrade/README.md

```diff
@@ -85,7 +85,7 @@ The upgrade is a guided process starting with [Upgrade Management Nodes and CSM
 
 After completion of the validation of CSM health, the CSM product stream has been fully upgraded and
 configured. Refer to the `1.5 HPE Cray EX System Software Getting Started Guide S-8000`
```

Comment:
Should "1.5" be included here? For Papaya, the guide is version 22.11 or edition 5. Is this a reference to an older version of the guide that should be removed?

```suggestion
configured. Refer to the `HPE Cray EX System Software Getting Started Guide S-8000`
```